detox/local-cli/testCommand/builder.js

```diff
@@ -28,6 +28,11 @@ module.exports = {
     describe: 'Reuse existing installed app (do not delete + reinstall) for a faster run.',
     boolean: true,
   },
+  start: {
+    group: 'Execution:',
+    describe: 'Run app "start" scripts before running the tests. Use --no-start to disable that, and --start=force to ignore errors.',
+    default: true,
```

Comment:
Debatable. Should the `--start` option be `true` by default?

Reply:
It should, IMO. If someone already defined the `start` property, so most-likely the `start` is necessary for the `test` command to work properly, and I can think of more cases where it will be more convenient to have it `true` by default than `false`.

Reply:
Also, we have a CLI option to disable the automatic running of `start`. Maybe we should also introduce another property for that

```diff
 {
   type: '...',
   binaryPath: '...',
   build: '...',
   start: 'react-native start', // ← for example
+  skipStartOnTest: true
 }
```

Reply:
(or just `skipStart: ...`)

Reply:
not sure if the last suggestion is really necessary, up to you